src/bic/collection/form-records.js

```diff
@@ -33,14 +33,14 @@ define(function (require) {
 
             collection.reset();
 
-            nodes = data.evaluate('//' + formName, data);
+            nodes = data.evaluate('//' + formName, data, null, XPathResult.ANY_TYPE, null);
             node = nodes.iterateNext();
 
             parseNodes = function (key) {
               if (key.nodeName === 'id') {
-                parsed.id = key.innerHTML;
+                parsed.id = key.textContent;
```

Comment:
this is what comes from getFormRecord and getFormList, it doesn't have new lines for xml tags, only if the data contains new line characters that will come up as new line. and textContent is what comes from DB so I don't think we can change it either.